go.mod

```diff
@@ -4,6 +4,7 @@ go 1.12
 
 require (
 	github.com/BurntSushi/toml v0.3.1
+	github.com/containers/image/v5 v5.4.1
```

Comment:
Does this imply the dependency graph being {image, storage}→{Skopeo,Buildah→Podman} ?

Works for c/image, I just want to make sure this is discussed.

Reply:
Yes, that's the idea. We need c/image and then c/storage to be at the end of our food chain. This way, we can move more things from libpod over here in the future.